dags/test_docker_failure.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+from pendulum import datetime, duration, now
+
+from utils.slack_operator import task_fail_slack_alert, slack_member_ids
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+DEFAULT_ARGS = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 1, 1, tz="America/Chicago"),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "execution_timeout": duration(minutes=5),
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+with DAG(
+    dag_id=f"test_docker_failure",
+    description="Throws a python exception from within a docker container",
+    default_args=DEFAULT_ARGS,
+    schedule_interval=None,
+    tags=["repo:atd-airflow", "slack"],
+    catchup=False,
+) as dag:
+    dag.byline = f"{slack_member_ids['Frank']} test failure"
```

Comment:
I think that I'd feel better without the mention here so we don't 📣 you when testing.

Reply:
I hear that Mike, and I can take it out -- I guess the rational that I left it in there was a reference usage of the dict of names, but we have another real one out there now to look at in the cris import dag. I'll remove this one.

Reply:
It's outta there! Thank you for this suggestion! https://github.com/cityofaustin/atd-airflow/pull/270/commits/d0e756138edbf276ffb3be67fbc4d55485946b23